core/location/interface.go

```diff
@@ -17,14 +17,9 @@
 
 package location
 
-import (
-	"net"
-)
-
 // Resolver allows resolving location by ip
 type Resolver interface {
 	DetectLocation() (Location, error)
-	ResolveLocation(ip net.IP) (Location, error)
```

Comment:
It was kept for cases when a consumer wants to check the provider's IP, whether it's really `residential` or not.
We, probably, don't another place for that now.